cost/google/idle_persistent_disk_recommendations/google_idle_persistent_disk_recommendations.pt

```diff
@@ -339,63 +340,71 @@ script "js_disk_cost_mapping", type:"javascript" do
+    })
+
+    _.each(recommendations, function (recommendation) {
+        count ++
+        var rec_resource_link = recommendation.resourceLink.replace("//compute.googleapis.com/", "")
+        var disk = _.find(disks, function(d) {
+            return d.resourceLink == rec_resource_link
+        })
+        if (!(disk == undefined || disk == null)) {
+            if (recommendation['currency'] !== undefined) {
+                if (recommendation['costUnits'] == null) {
+                    recommendation['costUnits'] = 0
                 }
-                if (recommendation.costNanos < 0) {
-                    recommendation.costNanos = recommendation.costNanos * -1
+                if (recommendation['costNanos'] == null) {
+                    recommendation['costNanos'] = 0
                 }
-                if (recommendation.costUnits < 0) {
-                    recommendation.costUnits = recommendation.costUnits * -1
+                if (ds_currency_reference[recommendation['currency']] !== undefined) {
+                    var cur = ds_currency_reference[recommendation['currency']]['symbol']
+                } else {
+                    var cur = "$"
                 }
-                var strNano = "." + recommendation.costNanos
-                var combine = recommendation.costUnits + parseFloat(strNano)
-                var monthsDuration = parseFloat(recommendation.duration.replace("s", "")) / 2628288
-                var monthlySavings = combine / monthsDuration
-                total = total + monthlySavings
-                disk['savings'] = (Math.round(monthlySavings * 1000) / 1000)
                 disk['savingsCurrency'] = cur
-                disk['description'] = recommendation['description']
-                disk['priority'] = recommendation['priority']
-                disk['primaryImpact'] = recommendation['primaryImpact']
-                disk['recommenderSubtype'] = recommendation['recommenderSubtype']
-                disk['state'] = recommendation['state']
-                disk['service'] = "Storage"
-                var tags = []
-                if (disk['tags'] != null) {
-                    if (typeof disk['tags'] == 'object') {
-                        Object.keys(disk.tags).forEach(function (key) {
-                            if (disk.tags[key] == null || disk.tags[key] == "") {
-                                disk.tags[key] = "null"
-                            }
-                            tags.push(String(key) + '=' + String(disk.tags[key]))
-                        });
-                    } else {
-                        tags.push(disk["tags"].replace(":", "=null"))
-                    }
+            } else {
+                var cur = "$"
+                var separator = ","
+            }
+            if (recommendation.costNanos < 0) {
+                recommendation.costNanos = recommendation.costNanos * -1
+            }
+            if (recommendation.costUnits < 0) {
```

Comment:
```suggestion
            if (recommendation.costUnits != 0) {
```

Reply:
@nia-vf1  thank you for this, I applied this change :)